roles/ceph-mon/templates/ceph-mon.service.j2

```diff
@@ -23,8 +23,12 @@ ExecStart=/usr/bin/docker run --rm --name ceph-mon-%i --net=host \
    --net=host \
    {% endif -%}
    -e CEPH_DAEMON=MON \
-   -e MON_IP={{ hostvars[inventory_hostname]['ansible_default_' + ip_version]['address'] }} \
+   {% if ip_version == 'ipv4' %}
+   -e MON_IP={{ hostvars[inventory_hostname]['ansible_' + ceph_mon_docker_interface][ip_version]['address'] }} \
    -e CEPH_PUBLIC_NETWORK={{ ceph_mon_docker_subnet }} \
+   {% elif ip_version =='ipv6' %}
+   -e MON_IP={{ hostvars[inventory_hostname]['ansible_' + ceph_mon_docker_interface][ip_version][0]['address'] }} \
```

Comment:
This is why I put this PR with "DNM" in the title actually, I want to find a way to be sure to take the address we want, maybe something like this: 
https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-common/templates/ceph.conf.j2#L35
could be ok but I need to make some tests to be sure it could work in ipv6.